src/overlays/actors/ovl_Boss_03/z_boss_03.c

```diff
@@ -41,157 +290,2352 @@ const ActorInit Boss_03_InitVars = {
+                            }
+                    }
+            }
+            break;
+
+        case 3:
+            Boss03_PlayUnderwaterSfx(&this->actor.projectedPos, NA_SE_EN_KONB_PREATTACK_OLD - SFX_FLAG);
+            sp5A = 0x1970;
+            Math_ApproachF(&this->actor.speedXZ, 15.0f, 1.0f, 2.0f);
+            if (this->csTimer > 20) {
+                this->csState = 4;
+                this->csTimer = 0;
+            }
+            break;
+
+        case 4:
+            player->actor.world.rot.y = player->actor.shape.rot.y = 0;
+
+            if (this->csTimer == 5) {
+                // Rotates Player towards Gyorg
+                func_800B7298(globalCtx, &this->actor, 8);
+            }
+
+            this->csCamEye.x = player->actor.world.pos.x + 30.0f;
+            this->csCamEye.y = player->actor.world.pos.y + Player_GetHeight(player) - 4.0f + BREG(17);
+            this->csCamEye.z = player->actor.world.pos.z - 30.0f;
+
+            this->csCamAt.x = player->actor.world.pos.x;
+            this->csCamAt.y = player->actor.world.pos.y + Player_GetHeight(player) - 18.0f + 6.0f + BREG(18);
+            this->csCamAt.z = player->actor.world.pos.z;
+
+            if (player->transformation == PLAYER_FORM_FIERCE_DEITY) {
+                this->csCamEye.y -= 60.0f;
+                this->csCamAt.y -= 35.0f;
+            }
+
+            this->cameraFov = 60.0f;
+            if (this->csTimer == 16) {
+                this->csState = 5;
+                this->csTimer = 0;
+                this->unk_2D5 = false;
+                this->actor.speedXZ = -200.0f;
+                Actor_MoveWithoutGravityReverse(&this->actor);
+                this->actor.world.pos.y = this->waterHeight - 150.0f;
+                func_80165690();
+
+                case 5:
+                    SkelAnime_Update(&this->skelAnime);
+                    this->actor.speedXZ = 20.0f;
+                    Actor_MoveWithoutGravityReverse(&this->actor);
+                    player->actor.shape.rot.y = -0x1470;
+                    player->actor.world.rot.y = player->actor.shape.rot.y;
+
+                    this->csCamEye.x = player->actor.world.pos.x + 30.0f - 90.0f + 300.0f;
+                    this->csCamEye.y = player->actor.world.pos.y + 40.0f + 10.0f;
+                    this->csCamEye.z = player->actor.world.pos.z - 30.0f + 160.0f + 300.0f;
+
+                    this->csCamAt.x = this->actor.world.pos.x;
+                    this->csCamAt.y = this->actor.world.pos.y - 100.0f;
+                    this->csCamAt.z = this->actor.world.pos.z;
+
+                    if (this->csTimer == 10) {
+                        this->actor.velocity.y = 30.0f;
+                        this->csState = 6;
+                        this->csTimer = 0;
+                        this->actor.gravity = -1.5f;
+                        this->actor.speedXZ = 20.0f;
+
+                        // Play the boss background music
+                        Audio_QueueSeqCmd(0x801B);
```

Comment:
```suggestion
                        Audio_QueueSeqCmd(NA_BGM_BOSS | 0x8000);
```
You might be able to delete the comment above this (and other places like it) once you do this, since it's more obvious what music plays now. Up to you, though.